src/main/java/io/jenkins/plugins/synopsys/security/scan/extension/pipeline/SecurityScanStep.java

```diff
@@ -34,10 +34,10 @@ public class SecurityScanStep extends Step implements Serializable {
     private String blackduck_url;
     private transient String blackduck_token;
     private String blackduck_install_directory;
-    private Boolean blackduck_scan_full;
+    private Boolean blackduck_scan_full = false;
```

Comment:
I think we don't need to set default value here if we remove the conditional part from setter method. Can you check please? I have checked and it's working for me.

Also if we set default value from our plugin then it will be printed as user provided value in the logs, but if the use doesn't pass any value why should we print the default value as user provided value that was once Tithi's concern.

Reply:
Yes @zaman-akib you are right, we do not need to pass default value here, I will change this setter methods for all the parameters. But we will do see these values being mentioned in the pipeline which was Tithi's concern.